launcher/src/main/java/org/apache/spark/launcher/SparkAppHandle.java

```diff
@@ -114,7 +114,7 @@ public boolean isFinal() {
    *
    * @since 1.6.0
    */
-  public interface Listener {
```

Comment:
To fix:
<img width="942" alt="image" src="https://github.com/apache/spark/assets/15246973/53020b2d-5dc6-4d63-9d20-bd49626ab79e">